cmd/processor/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+// move this functionality to cmd or a makefile
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"go-service-example/pb"
+	"go-service-example/pkg/processor"
+
+	kitgrpc "github.com/go-kit/kit/transport/grpc"
+	"github.com/oklog/oklog/pkg/group"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	grpcAddr := ":8081"
```

Comment:
it's no mandatory but, it would be fine if this "config" value can be extracted from an environment variable or configuration file like a .toml or .yaml